src/styles/variables.css

```diff
@@ -0,0 +1,3 @@
+:root {
+  --primary-color: #00acdc;
```

Comment:
I'm not quite sure what the purpose of this is

Reply:
@kamal-tufekcic google it

Reply:
@kamal-tufekcic  which class was not used? All the 3 classes have been used as follows:

![image](https://user-images.githubusercontent.com/109616961/187049039-9b2cba8c-2947-40e8-a3b0-922896d28745.png)
![image](https://user-images.githubusercontent.com/109616961/187049097-069111e1-62ee-4cce-8c3f-114d2f430c06.png)
![image](https://user-images.githubusercontent.com/109616961/187049135-b9b5349b-8fd8-4b0f-9098-705767d9f692.png)

It only looks like I have too many classes because I added a wrapper in order to give external styles to the card. I see you made that mistake on you code below. BEM rules don't allow us to add width to blocks like this.

![image](https://user-images.githubusercontent.com/109616961/187049216-905dde13-2bba-419b-b935-861960fb7093.png)

Reply:
@kamal-tufekcic oh I see what you mean the .card is a wrapper, I might use it for padding on the next task when I have all 8 cards in it. Without it I was gonna have to add the 200px width to the ".product" block. I saw you actually made that mistake lol.

Reply:
BEM slaps😂